index.html

```diff
@@ -10,5 +10,35 @@
     <header class="header">
         <h1>Hello world</h1>
     </header>
+    <form action="" method="post">
+        <ul>
```

Comment:
It's preferable to use divs instead of lists elements because you are not allowed to use <hr> or <br> in between them, and divs are by default block elements while list items are inline elements by default ^^